Kernel/VM/MemoryManager.cpp

```diff
@@ -1121,7 +1121,13 @@ void MemoryManager::unregister_region(Region& region)
 void MemoryManager::dump_kernel_regions()
 {
     dbgln("Kernel regions:");
-    dbgln("BEGIN         END        SIZE       ACCESS NAME");
+#if ARCH(I386)
+    auto addr_padding = "";
+#else
+    auto addr_padding = "        ";
```

Comment:
Sidenote: I wonder if using `StringView` literals (`"        "`) would avoid a runtime `strlen()` in cases like this. Not terribly important ofc.